tools/notcurses.spec

```diff
@@ -1,49 +1,109 @@
-Name: notcurses
-Version: 1.2.5
-Release: 1
-Summary: Character graphics and TUI library
-License: ASL 2.0
-Source0: https://github.com/dankamongmen/%{name}/archive/v%{version}.tar.gz
-Source1: https://github.com/dankamongmen/%{name}/releases/v%{version}/v%{version}.tar.gz.asc
-Source2: https://dank.qemfd.net/dankamongmen.gpg
-BuildRequires: gnupg2 cmake make gcc-c++ ncurses-devel pandoc python3-devel
+Name:          notcurses
+Version:       1.2.6
+Release:       1%{?dist}
+Summary:       Character graphics and TUI library
+License:       ASL 2.0
+URL:           https://nick-black.com/dankwiki/index.php/Notcurses
+Source0:       https://github.com/dankamongmen/%{name}/archive/v%{version}.tar.gz
+Source1:       https://github.com/dankamongmen/%{name}/releases/v%{version}/v%{version}.tar.gz.asc
+Source2:       https://dank.qemfd.net/dankamongmen.gpg
+
+BuildRequires: gnupg2
+BuildRequires: cmake
+BuildRequires: gcc-c++
+BuildRequires: pandoc
+BuildRequires: python3-devel
+BuildRequires: pkgconfig(ncurses)
```

Comment:
ahhh neat thank you; i'd missed https://fedoraproject.org/wiki/PackagingDrafts/PkgConfigBuildRequires